docs(common): Add KB for .NETFramework NuGet restore error (#1942)
telerik/blazor-docs: deployment/nuget-keys.md

* [Announcing NuGet Keys](https://www.telerik.com/blogs/announcing-nuget-keys)

> When restoring NuGet packages through CLI, make sure to [use `dotnet restore` rather than `nuget restore` in order to avoid compatibility errors]({%slug common-kb-package-telerik-pivot-not-compatible-with-netframework%}).

Reviewer: ```suggestion
When restoring NuGet packages through the CLI, [use `dotnet restore` rather than `nuget restore` to avoid compatibility errors]({%slug common-kb-package-telerik-pivot-not-compatible-with-netframework%}).
```
